presto-native-execution/pom.xml

```diff
@@ -49,6 +49,18 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-tpch</artifactId>
```

Comment:
Do we still need these changes?

Reply:
Turns out we need some others, just updated the dependencies according to the maven style check report.